src/core/async.ts

```diff
@@ -6,3 +6,7 @@ export async function findAsync<T>(arr: Array<T>, asyncCallback: (item: T) => Pr
 
 export const mapAsync = async <T, R>(arr: Array<T>, asyncCallback: (item: T) => Promise<R>): Promise<Array<R>> =>
 	Promise.all(arr.map(asyncCallback))
+
+export async function wait(delay: number) {
```

Comment:
ah, I thought I pushed the `delay` function, but it seems it only exists in my branch 😅